src/Cronjob/SendAlerts.php

```diff
@@ -130,7 +130,22 @@ public function do(bool $blnUpdateAlertLastJob = true): void
                 if ($objConditions && 0 < $objConditions->count()) {
                     while ($objConditions->next()) {
                         if ($objConditions->value) {
-                            $arrConditions[$objConditions->field] = $objConditions->value;
+                            $arrValue = unserialize($objConditions->value);
```

Comment:
According to https://www.php.net/manual/en/function.unserialize.php,

> In case the passed string is not unserializeable, [false](https://www.php.net/manual/en/reserved.constants.php#constant.false) is returned and [E_WARNING](https://www.php.net/manual/en/errorfunc.constants.php#constant.e-warning) is issued.

And

> Warning
> [false](https://www.php.net/manual/en/reserved.constants.php#constant.false) is returned both in the case of an error and if unserializing the serialized [false](https://www.php.net/manual/en/reserved.constants.php#constant.false) value. It is possible to catch this special case by comparing data with serialize(false) or by catching the issued [E_NOTICE](https://www.php.net/manual/en/errorfunc.constants.php#constant.e-notice).